src/main/java/engine/drivers/WebDriverInterface.java

```diff
@@ -0,0 +1,9 @@
+package engine.drivers;
+
+import org.openqa.selenium.WebDriver;
+
+public interface WebDriverInterface {
+    void setWebDriverPath();
+
+    WebDriver getDriver();
```

Comment:
No need for empty line between the methods of interface

Reply:
@Fishgame1 Fixed.